priv/static/icon-svg/403.svg

```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<svg viewBox="0 0 759 740" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I confess that this is an area of non-expertise for me, but both the Github diff viewer and Chrome (which my computer has decided is the default tool for viewing SVG files) are telling me that something is invalid about a bunch of these icons.

When I open the icon itself in Chrome, I get this:
```
error on line 122 at column 34: Namespace prefix xlink for href on use is not defined
```

Adding the `xmlns` and `xmlns:xlink` back in fixes that.

It renders just fine in-context in both Firefox and Chrome though 🤯

Reply:
well that's silly lol. well i may as well put them back then...